lib/components/app/print-layout.tsx

```diff
@@ -93,9 +108,7 @@ class PrintLayout extends Component<Props, State> {
             </SpanWithSpace>
             <SpanWithSpace margin={0.25}>
               <Button bsSize="small" onClick={this._print}>
-                <IconWithText Icon={Print}>
-                  <FormattedMessage id="common.forms.print" />
-                </IconWithText>
+                <IconWithText Icon={Print}>{printVerb}</IconWithText>
```

Comment:
Instead of "Transit from [...]", maybe "Trip from [...]"?
![image](https://user-images.githubusercontent.com/62163307/224824190-c7cfa6df-bf84-48e3-a79d-05584c3116ae.png)

Reply:
The `summarizeQuery` function shows the "primary" mode, so the resulting text could read "Walk from..." or "Drive from...".